core/retries/src/main/java/software/amazon/awssdk/retries/DefaultRetryStrategy.java

```diff
@@ -95,7 +95,7 @@ public static AdaptiveRetryStrategy.Builder adaptiveStrategyBuilder() {
 
     static final class Standard {
         static final int MAX_ATTEMPTS = 3;
-        static final Duration BASE_DELAY = Duration.ofSeconds(1);
+        static final Duration BASE_DELAY = Duration.ofMillis(100);
```

Comment:
Bug fix, the value is 100ms [see here](https://github.com/aws/aws-sdk-java-v2/blob/e6e22fd66fd8567968f23aaad304eee3559a893d/core/sdk-core/src/main/java/software/amazon/awssdk/core/internal/retry/SdkDefaultRetrySetting.java#L52).